app/screens/auth/set_up_screen.tsx

```diff
@@ -9,6 +9,7 @@ import { ThemedText } from "@/components/theme/ThemedText";
 import { ThemedScrollView } from "@/components/theme/ThemedScrollView";
 import { TouchableOpacity, Image } from "react-native";
 import * as ImagePicker from "expo-image-picker";
+import { useRouter } from "expo-router";
```

Comment:
This isn't used as the navigation system has been updated, unsure if this is the case in master as well. Either way we should remove it or make sure it doesn't merge into master the way it is right now.